### DISCUSSION TITLE
alist 级别注入用户脚本

### DISCUSSION THREAD

**Discussion by KnIfER** [AlistGo/alist#6847] [Ideas]
我有一个绝妙想法，若是可以在 alist 级别注入用户脚本，修改网页内容

可以做很多定制化，而无需重新编译。

比如：自定义缩略图服务，从ffmpeg获取本地文件的缩略图。

甚至可以修改列表内容，聚合排序，批量打开，按照文件名中的标签，分类排序，
按照日期分组排序，这些都是常规文件管理器没有的。

**Comment by KnIfER** [comment]
可以参考 alist - encrypt 做法，代理修改alist的页面。

**Comment by KnIfER** [comment]
算了，原页面比较卡的，三万多个文件浏览器卡崩溃了……

**Comment by Simonzhang8** [comment]
自己改源码编译试试？
